docs/apps/lnw/es2k/es2k-lnw-underlay-lag.md

```diff
@@ -0,0 +1,378 @@
+- Follow steps mentioned in [Deploying P4 Programs for E2100](/guides/es2k/deploying-p4-programs) for bringing up IPU with a custom P4 package.
+Modify `load_custom_pkg.sh` with following parameters for linux_networking package:
+
+```text
+    sed -i 's/sem_num_pages = 1;/sem_num_pages = 25;/g' $CP_INIT_CFG
+    sed -i 's/lem_num_pages = 1;/lem_num_pages = 10;/g' $CP_INIT_CFG
+    sed -i 's/acc_apf = 4;/acc_apf = 8;/g' $CP_INIT_CFG
+    sed -i 's/allow_change_mac_address = false;/allow_change_mac_address = true;/g' $CP_INIT_CFG
+```
+
+- Download `IPU_Documentation` TAR file specific to the build and refer to `Getting Started Guide` on how to install compatible `IDPF driver` on host. Once an IDPF driver is installed, bring up SRIOV VF by modifying the `sriov_numvfs` file present under one of the IDPF network devices. Example as below
+
+  ```bash
+  echo 1 > /sys/class/net/ens802f0/device/sriov_numvfs
+  ```
+
+Notes about topology:
+
+- VMs are spawned on top of each VF. Each VF will have a port representor in ACC. P4 runtime rules are configured to map VFs and their corresponding port representors.
+- Each physical port will have a port representor in ACC. P4 runtime rules are configured to map VFs and their corresponding port representors.
+- Each physical port will have a corresponding APF netdev on HOST. Create port representors in ACC for each HOST APF netdev. These APF netdevs on HOST will receive unknown traffic for applications to act on.
+- All port representors should be part of an OvS bridge. Based on topology, this OvS bridge will just perform bridging.
+- TEP termination on a bond interface which is used to enable underlay connectivity for VxLAN traffic.
+- OvS bridges and VxLAN ports are created using ovs-vsctl command (provided by the networking recipe).
+- Bond is created using bonding driver in kernel.
+- All port representors are attached to OvS bridge using ovs-vsctl command.
+- This config has:
+  - 1 Overlay VF
+  - 1 Port representors in ACC for the 2 Overlay VF
+  - 2 physical ports
+  - 2 Port representors in ACC for the 2 physical ports
+  - 2 APF netdevs on HOST
+  - 2 Port representors in ACC for the 2 HOST APF netdevs
+  - TEP termination dummy interface
+- LAG should be configured on both the devices with the associated ports as LAG members using ip link command.
+
+System under test will have above topology running the networking recipe. Link Partner can have the networking recipe or legacy OvS or kernel VxLAN. Refer to the Limitations section in [Linux Networking for E2100](es2k-linux-networking.md) before setting up the topology.
+
+## Creating the topology
+
+Follow steps mentioned in [Running Infrap4d on Intel E2100](/guides/es2k/running-infrap4d.md) for starting `infrap4d` process and creating protobuf binary for `fxp-net_linux-networking-v2` p4 program.
```

Comment:
double check the link, it's throwing 404 error

Reply:
See comment on line 35.